### PR TITLE
speed up launch time

### PR DESCRIPTION
**Pull request by novain1102**
Re: #https://github.com/cocos/3d-tasks/issues/15532

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by SantyWang — APPROVED** [review]

File: platforms/minigame/common/engine/AssetManager.js [hunk 1]
```diff
@@ -150,14 +150,13 @@ function loadFont (url, options, onComplete) {
...
 function doNothing (content, options, onComplete) {
-    // exists(content, (existence) => {
-    //     if (existence) {
-    //         onComplete(null, content);
-    //     } else {
-    //         onComplete(new Error(`file ${content} does not exist!`));
-    //     }
-    // });
-    onComplete(null, content);
+    exists(content, (existence) => {
```
Comment (minggo) [inline platforms/minigame/common/engine/AssetManager.js:153]: Now this function has logic, is it possible to still call it `doNothing()`?

Comment (novain1102) [reply platforms/minigame/common/engine/AssetManager.js:153]: Not sure. I didn't modify the code here. I guess call `doNothing()` is to unify the naming? @holycanvas

Comment (SantyWang) [reply platforms/minigame/common/engine/AssetManager.js:153]: Some asset needs post processing after downloading the file. and most of asset does not need post processing, So the callback function is named `doNothing`.

Comment (minggo) [reply platforms/minigame/common/engine/AssetManager.js:153]: Ok.

**Review by minggo — APPROVED** [review]

**Review by PPpro — APPROVED** [review]

**Pull request merged**